src/modules/Providers/hooks/useProviderInventory.ts

```diff
@@ -104,6 +104,7 @@ export const useProviderInventory = <T>({
         updateInventoryIfChanged(newInventory, fieldsToAvoidComparing);
         handleError(null);
       } catch (e) {
+        updateInventoryIfChanged(undefined, []);
```

Comment:
Hmm, I'm not sure using existing hooks such as this one is good for us without refactoring and splitting it. Do you have plans to do so in follow-up PRs?

Reply:
Im ok with create a complete new one, and also in a new jira and separate pr, could you please take care of it?

Reply:
https://issues.redhat.com/browse/MTV-2343